src/view/edit-event-view.js

```diff
@@ -116,54 +153,85 @@ export default class EditEventView extends AbstractStatefulView {
         'time_24hr': true,
         defaultDate: this._state.dateTo,
         minDate: this._state.dateFrom,
-        onChange: this.#onStartDateChange,
+        onChange: this.#onEndDateChange,
       },
     );
   }
 
-  reset() {
-    this.updateElement(this.#event);
+  #getCheckedOfferIds() {
+    const checkedOffers = [];
+    this.element.querySelectorAll('.event__offer-checkbox:checked').forEach((offer) => checkedOffers.push(offer.dataset.id));
+    return checkedOffers;
```

Comment:
У метода `Array.from` есть возможность передать map-функцию. Поэтому это всё можно написать в одну строку

```suggestion
return Array.from(this.element.querySelectorAll('.event__offer-checkbox:checked'), offer => offer.dataset.id);
```